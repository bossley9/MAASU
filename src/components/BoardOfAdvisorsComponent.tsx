import React, { FC } from 'react'
import { Container } from 'components/Container'
import { TextBlock } from 'components/TextBlock'
import { EA } from 'components/EmailAnchor'
import { A } from 'components/Link'
import { useDocumentTitle } from 'hooks/meta'
import { RouteComponentProps } from '@reach/router'
import { BOA_APP_LINK } from 'constants/strings'

type Props = RouteComponentProps & {}

export const BoardOfAdvisorsComponent: FC<Props> = () => {
  useDocumentTitle('Board of Advisors')

  return (
    <Container>
      <TextBlock title="Board of Advisors">
        <p>
          The Board of Advisors is a group of volunteers (working professionals
          with 5+ years or more of experience) with knowledge of and interest in
          the APIDA Community. Advisors have prior experience working with or
          advising a student organization, and have a strong understanding of
          the Midwest APIDA experience. The board serves as a resource to advise
          the Directors Council (DC) and Executive Director (ED).
        </p>
        <p>
          For more information, please view our{' '}
          <A href={BOA_APP_LINK}>application form.</A>
        </p>
        <p>
          Thank you for your interest in being a volunteer for the MAASU Board
          of Advisors! If you have any questions, comments, or concerns, please
          contact the Executive Director at <EA>execdirector@maasu.org</EA>.
        </p>
      </TextBlock>
    </Container>
  )
}
