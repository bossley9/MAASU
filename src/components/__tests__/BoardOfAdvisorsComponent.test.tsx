import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import { BoardOfAdvisorsComponent } from '../BoardOfAdvisorsComponent'

describe('render', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<BoardOfAdvisorsComponent />, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

it('renders a matching snapshot', () => {
  const test = renderer.create(<BoardOfAdvisorsComponent />)
  expect(test.toJSON()).toMatchSnapshot()
})
