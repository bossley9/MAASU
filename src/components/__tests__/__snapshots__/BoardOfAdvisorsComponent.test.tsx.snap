// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a matching snapshot 1`] = `
<div
  className="container py6 "
>
  <div
    className="my4 "
  >
    <section
      className=""
    >
      <h4
        className="title4 tc my2"
      >
        Board of Advisors
      </h4>
      <p>
        The Board of Advisors is a group of volunteers (working professionals with 5+ years or more of experience) with knowledge of and interest in the APIDA Community. Advisors have prior experience working with or advising a student organization, and have a strong understanding of the Midwest APIDA experience. The board serves as a resource to advise the Directors Council (DC) and Executive Director (ED).
      </p>
      <p>
        For more information, please view our
         
        <a
          className="c-text-main tdu hov-c-primary-main "
          href="
https://firebasestorage.googleapis.com/v0/b/maasu-master.appspot.com/o/BoardMembers%2F20-21%2Fapp-boa.pdf?alt=media&token=a2d3751c-cab6-4bb3-8e73-0dbae5c0c782
"
          rel="noreferrer noopener"
          target="_blank"
        >
          application form.
        </a>
      </p>
      <p>
        Thank you for your interest in being a volunteer for the MAASU Board of Advisors! If you have any questions, comments, or concerns, please contact the Executive Director at 
        <a
          className="c-text-main tdu hov-c-primary-main "
          href="mailto:execdirector@maasu.org"
          rel="noreferrer noopener"
          target="_blank"
        >
          execdirector@maasu.org
        </a>
        .
      </p>
    </section>
  </div>
</div>
`;
